source/_patterns/01-elements/link/link.scss

```diff
@@ -26,42 +27,40 @@
 			color: initial;
 		}
 	}
+	&[data-variant="internal"] {
+		@include icon(glyph(link), 20, "outline", "after");
+	}
+	&[data-variant="external"] {
+		@include icon(glyph(link-external), 20, "outline", "after");
+	}
 
+	&[data-variant="internal"],
+	&[data-variant="external"] {
+		color: #282d37;
+		&::after {
+			margin-top: -3px;
+			--icon-margin-before: #{to-rem($pxValue: 4)};
+		}
+		&:focus-visible {
+			outline-offset: -1px;
+			padding: 0 2px 0 2px;
+		}
+	}
 	// Sizes
 	&%size-Small {
 		&:not(.is-icon-text-replace) {
-			font-size: to-rem($pxValue: 14);
-		}
-		&[rel],
-		&[data-icon],
-		&[data-icon-before] {
-			@include iconMeta(20);
+			font-size: to-em($pxValue: 12);
 		}
-		&[data-icon-after]::after {
-			@include iconMeta($size: 20, $position: "after");
+		&:active {
+			font-size: to-em($pxValue: 10);
 		}
 	}
 
 	&[data-size="small"] {
+		color: #42464f;
 		@extend %size-Small;
-	}
-
-	// Define the available icons
-	&[rel="configuration"] {
-		@include icon(glyph(settings), 24, "outline", $partial: $partial);
-	}
-	&[rel="messages"] {
-		@include icon(glyph(chat), 24, "outline", $partial: $partial);
-	}
-	&[rel="account"] {
-		@include icon(glyph(account), 24, "outline", $partial: $partial);
-	}
-	&[rel="configuration"],
-	&[rel="messages"],
-	&[rel="account"] {
-		&::before {
-			// preventing an underlined icon on click
-			display: inline-block;
```

Comment:
we would still need those for enterprise header, so I could think about moving these declarations to this subfolders CSS.

Reply:
Fixed